test/Streamly/Test/Internal/Data/Fold.hs

```diff
@@ -2,26 +2,175 @@ module Main (main) where
 
 import qualified Streamly.Prelude as S
```

Comment:
You might want to rearrange the imports. Maybe group the qualified imports togather?

Reply:
Sure.

Reply:
How about having `Streamly` imports together first and then `Test` imports?

```
import Streamly.Internal.Data.Fold
import qualified Streamly.Prelude as S
import qualified Streamly.Data.Fold as FL

import Test.Hspec as H
import Test.Hspec.QuickCheck
import Test.QuickCheck (Property, forAll, Gen, vectorOf, arbitrary, choose)
import Test.QuickCheck.Monadic (monadicIO, assert, run)

```

Reply:
It is nice to have qualified imports as a seperate block.